lib/order.rb

```diff
@@ -0,0 +1,36 @@
+require './lib/send_sms'
+
+class Order
+
+  attr_reader :menu
+
+  def initialize
+    @order = []
+  end
+
+  def add_menu(menu)
+    @menu = menu
+  end
+
+  def add_dish(dish)
+    @order.append(dish.name => dish.price)
+  end
+
+  def contents
+    @order
+  end
+
+  def total
```

Comment:
Maybe the Order class is starting to do too many things? Could Total be a separate class?